### PR TITLE
Load supported entities in base class

### PR DESCRIPTION
**Pull request by akshaybhalotia**

### CODE REVIEW

**Comment by akshaybhalotia** [comment]
Fixes #20

File: CHANGELOG [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Changelog for Razorpay-Ruby SDK.
 
 ## Unreleased
+### Added
+- Adds `require` for all Razorpay supported entities
+
+## [1.2.1] - 2106-12-22
```
Comment (amitmahbubani) [inline CHANGELOG]: typo, 2016

File: lib/razorpay/constants.rb [hunk 2]
```diff
@@ -2,5 +2,5 @@
 module Razorpay
   BASE_URI = 'https://api.razorpay.com/v1/'.freeze
   TEST_URL = 'https://api.razorpay.com/'.freeze
-  VERSION = '1.2.0'.freeze
+  VERSION = '1.2.2'.freeze
```
Comment (captn3m0) [inline lib/razorpay/constants.rb]: @akshaybhalotia Never bump version in a feature-PR. You don't want a situation where another PR bumps it as well. The release follows what we have here: https://github.com/razorpay/razorpay-php#release

Comment (akshaybhalotia) [inline lib/razorpay/constants.rb]: Sure, will take care of this. This info wasn't available on this repo, hence the mistake.

Comment (captn3m0) [inline lib/razorpay/constants.rb]: I'm reverting this back to 1.2.1 for now till we have a new release.

**Pull request merged**